pom.xml

```diff
@@ -3,15 +3,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.57</version>
+        <version>4.31</version>
         <relativePath />
     </parent>
     <packaging>hpi</packaging>
     <properties>
         <revision>1.25</revision>
         <changelist>-SNAPSHOT</changelist>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.303.1</jenkins.version>

```

Comment:
Just a note for myself. This is an LTE relase 5 months old

Reply:
We can drop it down a little with a matching plugin bom